src-admin/src/Tabs/Bridges.tsx

```diff
@@ -1156,9 +1176,9 @@ export class Bridges extends BridgesAndDevices<BridgesProps, BridgesState> {
         devIndex: number,
     ): React.JSX.Element {
         const isLast = devIndex === bridge.list.length - 1;
-        const hasError = Array.isArray(this.props.nodeStates?.[bridge.uuid]?.error)
-            ? !!(this.props.nodeStates[bridge.uuid].error as string[])?.includes(device.uuid)
-            : false;
+        const errorInfo = this.props.nodeStates?.[bridge.uuid]?.error;
+        const hasError =
+            typeof errorInfo !== 'boolean' && Array.isArray(errorInfo) ? errorInfo.includes(device.uuid) : false;
```

Comment:
`typeof errorInfo !== 'boolean'` is not required here `Array.isArray(errorInfo)` is enought

Reply:
not for typescript :-) With this additional check also TS is happy and do not require a type hint